c_cxx/CMakeLists.txt

```diff
@@ -27,16 +23,25 @@ set(CMAKE_CXX_STANDARD 17)
 
 if(NOT ONNXRUNTIME_ROOTDIR)
   if(WIN32)
-    set(ONNXRUNTIME_ROOTDIR "C:/Program Files (x86)/onnxruntime")
-  else()
-    include_directories("/usr/local/include/onnxruntime")
+    set(ONNXRUNTIME_ROOTDIR "C:/Program Files/onnxruntime")
+else()
+    set(ONNXRUNTIME_ROOTDIR "/usr/local")
   endif()
 endif()
 
+# The ORT package has a different include directory structure to a local install via cmake.
+# We added the path for the pre-built package above. Add the path for a local install to support either usage.
+# TODO: If we want to support additional EPs being loadable from a local install we also need to add EP specific
+# directories under /include/onnxruntime/core/providers
```

Comment:
would this be needed if the onnxruntime_USE_CUDA or onnxruntime_USE_TENSORRT options are set?